core/src/main/scala/org/apache/spark/rdd/RDD.scala

```diff
@@ -1780,10 +1780,9 @@ abstract class RDD[T: ClassTag](
    * It will result in new executors with the resources specified being acquired to
    * calculate the RDD.
    */
-  // PRIVATE for now, added for testing purposes, will be made public with SPARK-29150
   @Experimental
```

Comment:
Thanks for adding this issue. I was about to suggest to keep it same `Evolving` but I suspect there's a reason for it :-). LGTM

Reply:
this one I just missed originally